### ISSUE TITLE
keyword matched when that word is a sub-string of another word

### ISSUE THREAD

**Issue by wangx173** [UTMediaCAT/Voyage#41]
Example:
if the keyword is "ISIS",

cr**isis**,  Datanál**isis** will be considered as match

**Comment by wangx173** [comment]
http://159.203.7.224/admin/articles/article/326/    

check it out

**Comment by Sunakujira1** [comment]
As I said, I cannot find a single url from your server that I can recreate the problem.
```python
>>> c = ExplorerArticle("http://www.bbc.com/mundo/noticias/2015/12/151127_america_latina_albinos_puerto_rico_aislamiento_nieves_amv")
>>> c.download()
True
>>> c.preliminary_parse()
True
>>> regex = re.compile('[^a-z]' + 'isis' + '[^a-z]', re.IGNORECASE)
>>> regex.search(c.text)
>>> regex.search(c.title)
>>>
```

**Comment by wangx173** [comment]
that's weird, cuz I am running it on master with the most recent commit, can you try it on another server?

**Comment by Sunakujira1** [comment]
Nevermind, fixed the issue at 543989c73212bfb44b0af871b93d67f2dcd6781b
